client/web/compose/src/components/PageBlocks/RecordListConfigurator.vue

```diff
@@ -998,6 +986,13 @@ export default {
     isInlineEditorAllowed () {
       return this.recordListModule && (this.onRecordPage || this.options.editable)
     },
+
+    inlineEditButtonTooltip () {
```

Comment:
Its fine if you define the tooltip  in the template itself (since it like this for other places)